app/src/main/res/values/strings.xml

```diff
@@ -159,14 +159,16 @@
     <string name="edit_gender">Edit</string>
     <string name="update_gender">Update Gender</string>
     <string name="menu_refresh">Refresh</string>
-    <string name="policy">By clicking on Log In, you agree with our Terms. Learn how we process you data in our Privacy Policy and Cookies Policy.</string>
+    <string name="policy">By clicking on Sing In, you agree with our Terms. Learn how we process your data in our Privacy Policy and Cookies Policy.</string>
```

Comment:
Small typo here :P (unless maybe you expect people to sing in their recording? xD) 
```suggestion
    <string name="policy">By clicking on Sign In, you agree with our Terms. Learn how we process your data in our Privacy Policy and Cookies Policy.</string>
```

Reply:
Oops, fixing typo with another typo :D